CoreFoundation/String.subproj/CFString.h

```diff
@@ -165,7 +165,7 @@ struct __CFConstStr {
         uint8_t _pad[4];
     } _base;
     uint8_t *_ptr;
-    uint32_t _length;
+    unsigned long _length;
```

Comment:
The aliasing of a 32-bit _length field with a 64-bit CFIndex field will not work on any 64-bit big-endian system (e.g. s390x and powerpc64). So I believe the guard should be:

```
#if defined(__LP64__) && defined(__BIG_ENDIAN__)
```